vault/logical_system.go

```diff
@@ -1709,7 +1709,16 @@ func (b *SystemBackend) handlePolicyRead(
 func (b *SystemBackend) handlePolicySet(
 	req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
 	name := data.Get("name").(string)
-	rules := data.Get("rules").(string)
+
+	rulesRaw, ok := data.GetOk("rules")
+	if !ok {
+		return logical.ErrorResponse("missing rules input"), nil
```

Comment:
I'd make this more explicit that it's not just that they're missing rules, it's that they're missing the rules *parameter*. Maybe something like `'rules' parameter not supplied`

Same comment below.